parsec/_parsec_pyi/protocol.pyi

```diff
@@ -0,0 +1,64 @@
+from typing import Optional
+from parsec._parsec_pyi.ids import BlockID, RealmID
+
+class AuthenticatedAnyCmdReq:
+    def __repr__(self) -> str: ...
+    def dump(self) -> bytes: ...
+    def load(buf: bytes) -> BlockReadReq: ...
+
+class InvitedAnyCmdReq:
+    def __repr__(self) -> str: ...
+    def dump(self) -> bytes: ...
+    def load(buf: bytes) -> BlockReadReq: ...
+
+class BlockCreateReq:
+    def __init__(self, block_id: BlockID, realm_id: RealmID, block: bytes) -> None: ...
+    def __repr__(self) -> str: ...
+    def __eq__(self, other: BlockCreateReq) -> bool: ...
```

Comment:
In which case do we need `__eq__/__ne__` operators ? (I guess this is for tests)

Reply:
Yes for tests